example/src/App.tsx

```diff
@@ -90,6 +93,37 @@ function App() {
         }
     }
 
+    function requestAccounts() {
+        setError('')
+        const provider = window.splice
+
+        if (provider !== undefined) {
+            provider
+                .request({
+                    method: 'requestAccounts',
+                })
```

Comment:
I was thinking it makes more sense to put this inside the `useEffect` above (like how it calls the `status` method). 

That'll call the method automatically when the component first renders, so we can update the primary party immediately (if we have existing wallets). The provider (& auth token) is cached, so this also refetches the data if the user refreshes the page